### ISSUE TITLE
As reviewer, I can edit, correct or extend records so they can be validated [Review workflow case]

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#353]
# Story

As reviewer, I can edit, correct or extend records so they can be validated.

TLDR;
- Initial status: Published & In review
- Actions: Reviewer can view and edit new published records and inform researchers about the changes.
- Eventual status: Published & Validated

# Success criteria

## Status (same as #351)
- - [ ] New, published records added by researchers get the visibility status (Biblio status): Published
- - [ ] New, published records added by researchers get the review status: In review

## Workflow (after #351)
- - [ ] The reviewer can skim all fields, and correct them if needed.
- - [ ] [TBD – ONLY IF NEEDED] The reviewer motivates why these corrections took place.

### If record complete:
- - [ ] The record keeps the visibility status: Published
- - [ ] The reviewer **changes the review status to: Validated**
- - [ ] The record is automatically locked after validation from any other adaptions by the researcher.

#### Automatic updates
- - [ ] [IN PROGRESS] After changing the review status to Validated, the involved researcher(s) get an email and/or notification stating their record is complete and has been validated.
- - [ ] If there have been made any adaptions to the record, the involved researcher(s) also, in that same email and/or notification, get an update about their changes and why they have been changed.

### If record incomplete:
- If the record still has faulty or incomplete fields (after adaption) go to workflow:
- - Missing non-breaking information: #354.
- - Missing important, breaking information: #355.
- If the record appears to have damaging information, go to workflow #356.

## Visibility (remains the same as #351)
- - [ ] The record remains visible to involved researchers (? TBD: or only draft for editor itself?) in their publications
- - [ ] The record remains visible to involved researchers on their resumé
- - [ ] The record remains visible on the grant summary page
- - [ ] The record remains visible in Biblio, available to the rest of the world
- - [ ] The record keeps the persistent URL (handle) that can be (re)used by professional instances

# Additional info

This story is part of epic #347, presented as one of the cases that fit into the review workflow that involves motivating researchers to complete records.

**Comment by mietcls** [comment]
Will rethink
